### ISSUE TITLE
Deserialization Error?

### ISSUE THREAD

**Issue by austinleekelly** [yannick-cw/notionfy#15]
```
Unfortunately syncing failed, you can run with --verbose to see more details about what is going on. Please report the output to https://github.com/yannick-cw/notionfys, so I can fix it. Here is the raw error:
sttp.client.DeserializationError
notionfys.Notion$.$anonfun$getHighlights$3(Notion.scala:80)
scala.util.Either$LeftProjection.map(Either.scala:614)
notionfys.Notion$.$anonfun$getHighlights$2(Notion.scala:80)
cats.data.Kleisli.$anonfun$flatMap$2(Kleisli.scala:43)
cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:139)
cats.effect.internals.IORunLoop$RestartCallback.signal(IORunLoop.scala:355)
cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:376)
cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:316)
cats.effect.internals.IOShift$Tick.run(IOShift.scala:36)
cats.effect.internals.PoolUtils$$anon$2$$anon$3.run(PoolUtils.scala:51)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
java.lang.Thread.run(Thread.java:834)
com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:527)
com.oracle.svm.core.windows.WindowsJavaThreads.osThreadStartRoutine(WindowsJavaThreads.java:138)
```

**Comment by yannick-cw** [comment]
Thanks I'll look into that!

**Comment by austinleekelly** [comment]
Okay thanks! Really looking forward to having my highlights 😊😊😊

**Comment by joethekurian** [comment]
Fetching highlights from Notion...
Unfortunately syncing failed, you can run with --verbose to see more details about what is going on. Please report the output to https://github.com/yannick-cw/notionfys, so I can fix it. Here is the raw error:
sttp.client.DeserializationError
notionfys.Notion$.$anonfun$getHighlights$3(Notion.scala:80)
scala.util.Either$LeftProjection.map(Either.scala:614)
notionfys.Notion$.$anonfun$getHighlights$2(Notion.scala:80)
cats.data.Kleisli.$anonfun$flatMap$2(Kleisli.scala:43)
cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:139)
cats.effect.internals.IORunLoop$RestartCallback.signal(IORunLoop.scala:355)
cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:376)
cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:316)
cats.effect.internals.IOShift$Tick.run(IOShift.scala:36)
cats.effect.internals.PoolUtils$$anon$2$$anon$3.run(PoolUtils.scala:51)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
java.lang.Thread.run(Thread.java:834)
com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:527)
com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:193)

**Comment by austinleekelly** [comment]
Any updates?

**Comment by neon-ewok** [comment]
I'm having the same issue as OP 😭  I'd really love to get this up and running.  Any help available?

**Comment by yannick-cw** [comment]
I am sorry for the delay, I had some vacation and so on..
I released `0.2.6`, please see if it helps. Notion had an api change, which made notionfy fail. It should be fine now, otherwise please send me your verbose output here and I'll try.
